site/content/docs/5.1/components/alerts.md

```diff
@@ -155,7 +155,7 @@ Add `.bg-dark` to the `.alert` for a dark variant. Close button can be inverted
 
 ## CSS
 
-<small class="d-inline-flex px-2 py-1 font-monospace text-muted border rounded-3">Added in v5.2.0</small>
+<small class="d-inline-flex px-2 py-1 fw-semibold text-success bg-success bg-opacity-10 rounded-2">Added in v5.2.0</small>
```

Comment:
Bootstrap modified the appearance of this "Added in" information.

**Before in Bootstrap**
![Screenshot from 2022-03-29 11-19-18](https://user-images.githubusercontent.com/17381666/160581098-7ef4bc35-0061-4d19-8496-5957f5f7bfb0.png)
**

**Now in Bootstrap**
![Screenshot from 2022-03-29 11-18-14](https://user-images.githubusercontent.com/17381666/160581040-f274d4ad-416e-4b66-94cf-246d93e4d82a.png)

In Boosted, the current display is:
![Screenshot from 2022-03-29 11-20-10](https://user-images.githubusercontent.com/17381666/160581166-c689accb-1121-4f87-97c9-b9dd9e617a45.png)

What is proposed in this PR for Boosted is:
![Screenshot from 2022-03-29 11-21-25](https://user-images.githubusercontent.com/17381666/160581216-61080624-3b3a-4089-82ba-324bdfc5c8ef.png)

This proposal is based on the exact same code than Bootstrap.
@CyriaqueBillard @Franco-Riccitelli I know we are not too much allowed to write on green background and have rounded border.
Does it sound OK or should I have straight borders and change the background color to maybe light gray?

Reply:
Answers from design:
- [ ] Temporarily use `#ccc` as a background without border.
- [ ] In the future, use the Tags. Create an issue or comment the actual Tags PR in order to use them and to not to forget to do it.